client/src/components/footer/footer.js

```diff
@@ -139,30 +140,23 @@ class Footer extends Component {
                 }
             }
         }
-
-
     }
 
-
     render() {
         const {state, actions} = this.props;
-        let display = state.game.state === "GAME_IDLE" || state.app.body !== 2;
+        let display = (state.game.state === "GAME_IDLE" || state.app.body !== 2) && state.main.lab !== 0;
         return (
-
             <div>
                 <div className="container" style={{display: display ? "block" : "none"}}>
-
                     <button
                         className="btn btn-second btn-xl text-uppercase js-scroll-trigger back "
-                        onClick={() => this.handleDecrement(state.app.body, actions)
-                        }
+                        onClick={() => this.handleDecrement(state.app.body, actions)}
                         style={{display: this.disappearBack(state.app.body) ? "none" : "block"}}
```

Comment:
you can add logic here: display: this.disappearBack(state.app.body) **|| !displayBackNext**